resources/views/livewire/order/order-positions.blade.php

```diff
@@ -100,7 +100,13 @@ class="pb-4"
             <x-slot:footer>
                 <div class="flex justify-between gap-x-4">
                     <div x-show="$wire.orderPosition.id">
-                        <x-button flat negative :label="__('Delete')" x-on:click="$wire.deleteOrderPosition(); close();" />
+                        <x-button
+                            flat
+                            negative
+                            :label="__('Delete')"
+                            wire:click="deleteOrderPosition(); close();"
```

Comment:
wait this works? To call close(); afterwards?

Reply:
sure, wire:click is basically the same as x-on:click="$wire.doSomeThing", so you can add whatever javascript you want afterwards